### PR TITLE
Categories block

### PR DESCRIPTION
**Pull request by chigby**
Refs #1193

### CODE REVIEW

**Comment by SaptakS** [comment]
I was able to achieve something of these sorts:

image


Basically what I did was remove the `flex: 1` in the child elements which I guess you did so that each row takes up 100% space. And then to each of those elements, I added a `margin-left: -3px; margin-top: -3px` if the border width is 3px, to compensate for the double bordering. Then you can adjust the parent element's margin to compensate for the overall 3px shift to top and left. Also, the actual design being exactly same as the design would definitely depend on the order in which the categories are shown, which I think is fine. Also, you will need to add `justify-content: center` to the parent element.

**Comment by chigby** [comment]
@SaptakS I put the finishing touches on this one and it's ready for review. Let me know what you think.

**Comment by chigby** [comment]
Rebased this on the latest `category-symbol` branch. It's ready for review (and should be merged after that branch).

File: client/common/sass/components/_categories-block.sass [hunk 1]
```diff
@@ -0,0 +1,22 @@
+.categories-block
+	display: flex
+	flex-flow: row wrap
+	justify-content: center
+	list-style: none
+	margin: 72px 5rem
+	padding: 0
+
+.categories-block--item
```
Comment (SaptakS) [inline client/common/sass/components/_categories-block.sass]: This should be `&--item` instead.

**Review by SaptakS — CHANGES_REQUESTED** [review]
Only one small change. After that, this can be merged

File: client/common/sass/components/_categories-block.sass (hunk 1 above)
Comment (chigby) [reply client/common/sass/components/_categories-block.sass]: Good catch here, I've added it.

**Review by SaptakS — APPROVED** [review]

**Pull request merged**